examples/java/core/enumerations/src/main/bond/enumerations.bond

```diff
@@ -0,0 +1,21 @@
+namespace Examples;
+
+enum Color
+{
+    Orange = 1,
+    Yellow = 2,
+}
+
+enum Fruit
+{
+    Apple = 1,
+    Orange = 2,
+}
+
+enum Limits
+{
+    Int32Min = -2147483648,
+    Int32Max = 2147483647,
+    UInt32Min = 0,
+    UInt32Max = 0xFFFFFFFF,
```

Comment:
I don't agree with the example (IMO it's misleading) but I agree with the change since this is just a port. No issue.

---
In reply to: [132254217](https://github.com/Microsoft/bond/pull/551#discussion_r132254217) [](ancestors = 132254217)